### DISCUSSION TITLE
Translations and language strings

### DISCUSSION THREAD

**Discussion by nilsteampassnet** [nilsteampassnet/TeamPass#3580] [Show and tell]
Hello!

All language strings and translations are managed from [POEditor](https://poeditor.com/join/project?hash=0vptzClQrM).

In order to particpate, create your own POEditor account and participate to translate your favorite language strings.

New strings and translations are regularly synchronized with Teampass source files.

---
> Link has been edited with corrected URL

**Comment by mtwango** [comment]
The link doesn't seem to work.

**Reply by Kyogre** [reply]
https://github.com/nilsteampassnet/TeamPass/issues/4121

**Reply by mtwango** [reply]
Yeah found out, thx.
